src/allocator/memserver_allocator.h

```diff
@@ -75,15 +87,21 @@ class Memserver_Allocator {
     void *get_local_pointer(uint64_t regionId, uint64_t offset);
     void open_heap(uint64_t regionId);
     void create_ATL_root(size_t nbytes);
+    void remove_heap_from_list(uint64_t regionId);
+    void garbage_collector_th(uint64_t thread_index);
 
   private:
     MemoryManager *memoryManager;
+    EpochManager* em;
+    EpochOp* op;
```

Comment:
I dont think we need a "EpochOp* op" private variable.

Reply:
Need to see this.